### PR TITLE
Branch Operations 

### PR DESCRIPTION
**Pull request by VishnuPalissery**
A new branch is created and a file is added.

### CODE REVIEW

**Comment by VishnuPalissery** [comment]
updated..!

**Comment by AnithaVJ** [comment]
@VishnuPalissery Edit this title and you are good to go.

**Comment by VishnuPalissery** [comment]
ok..!

**Comment by AnithaVJ** [comment]
@VishnuPalissery Nope not this title. No need to put branch names in title, that's irrelevant.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Freshers-2018-August
 Git and GitHub familiarisation
-
+bioBranch
```
Comment (AnithaVJ) [inline README.md]: Where did this come from?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+First line added
 A new line inserted...
 -akshai
+develop
```
Comment (AnithaVJ) [inline README.md]: Same here. This was not present before conflicts.

**Comment by AnithaVJ** [comment]
:shipit:

**Pull request merged**